packages/workbox-routing/Route.mjs

```diff
@@ -82,14 +78,11 @@ class Route {
    *
    * @param {module:workbox-routing.Route~matchCallback} match
    * A callback function that determines whether the route matches a given
-   * `fetch` event.
-   * @param {
-   *   module:workbox-routing.Route~handlerCallback |
-   *   module:workbox-runtime-caching.Handler
-   * } handler This parameter can be either a callback function or an object
-   * which implements the `Handler` interface.
-   * @param {string} [method='GET'] Restrict the route to only match requests
-   * that use this HTTP method.
+   * `fetch` event by returning a non-falsy value.
+   * @param {module:workbox-routing.Route~handlerCallback} handler A callback
+   * function that returns a Promise resulting in a Response.
```

Comment:
"resulting in" => "resolves to" (That's the phrasing [used in MDN](https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch).)